executor/cook/progress.py

```diff
@@ -237,40 +268,31 @@ def retrieve_progress_states(self):
         -------
         an incrementally generated list of progress states.
         """
+        last_unprocessed_report = None
         if self.progress_regex_string:
             sleep_time_ms = 50
             for line in self.tail(sleep_time_ms):
                 try:
                     progress_report = ProgressWatcher.match_progress_update(self.progress_regex_pattern, line)
                     if progress_report is None:
```

Comment:
Rewrite this as "if/elif/else" instead of using continue

Reply:
Done.